services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -169,17 +173,32 @@ func (s PSMDBClusterService) UpdatePSMDBCluster(ctx context.Context, req *dbaasv
 			Kubeconfig: kubernetesCluster.KubeConfig,
 		},
 		Name: req.Name,
-		Params: &dbaascontrollerv1beta1.UpdatePSMDBClusterRequest_UpdatePSMDBClusterParams{
-			ClusterSize: req.Params.ClusterSize,
-			Replicaset: &dbaascontrollerv1beta1.UpdatePSMDBClusterRequest_UpdatePSMDBClusterParams_ReplicaSet{
-				ComputeResources: &dbaascontrollerv1beta1.ComputeResources{
-					CpuM:        req.Params.Replicaset.ComputeResources.CpuM,
-					MemoryBytes: req.Params.Replicaset.ComputeResources.MemoryBytes,
-				},
-			},
-		},
 	}
 
+	if req.Params != nil {
+		in.Params = new(dbaascontrollerv1beta1.UpdatePSMDBClusterRequest_UpdatePSMDBClusterParams)
+		in.Params.ClusterSize = req.Params.ClusterSize
+
+		if req.Params.Suspend && req.Params.Resume {
```

Comment:
I think it should be higher before initializing `in.Params`

Reply:
Moved.